test/arcade/potts/env/location/PlaneTest.java

```diff
@@ -159,4 +161,144 @@ public void hashCode_equalObjects_returnsSameCode() {
 
         assertEquals(plane1.hashCode(), plane2.hashCode());
     }
+
+    @Test
+    public void rotateNormalAroundAxis_Called_vectorMagnitudePreserved() {
+        Double3D normalVector = new Double3D(5, 0, 0);
+        Direction axis = Direction.XY_PLANE;
+        double thetaDegrees = 90.0;
+
+        Double3D rotatedVector = Plane.rotateVectorAroundAxis(normalVector, axis, thetaDegrees);
+
+        double magnitude = Plane.getVectorMagnitude(rotatedVector);
+        assertEquals(0.0, rotatedVector.x, EPSILON);
+        assertEquals(5.0, rotatedVector.y, EPSILON);
+        assertEquals(0.0, rotatedVector.z, EPSILON);
+        assertEquals(5, magnitude);
+    }
+
+    @Test
+    public void rotateNormalAroundAxis_thetaZero_returnsSameVector() {
+        Double3D normalVector = new Double3D(1, 0, 0);
+        Direction axis = Direction.XY_PLANE;
+        double thetaDegrees = 0.0;
+        Double3D rotatedVector = Plane.rotateVectorAroundAxis(normalVector, axis, thetaDegrees);
+        assertEquals(normalVector, rotatedVector);
+    }
+
+    @Test
+    public void rotateNormalAroundAxis_xAxis_returnsCorrectRotation() {
```

Comment:
For all of these test names, the method is `rotateVectorAroundAxis` and not `rotateNormalAroundAxis`?